webservices/resources/legal.py

```diff
@@ -0,0 +1,30 @@
+import sqlalchemy as sa
+
+from flask_apispec import doc, marshal_with
+
+from webservices import args
+from webservices import docs
+from webservices import utils
+from webservices.utils import use_kwargs
+from webargs import fields
+from pyelasticsearch import ElasticSearch
+from webservices.env import env
+
+
+@doc(
+    tags=['search'],
+    description=docs.LEGAL_SEARCH
+)
+
+class UniversalSearch(utils.Resource):
+    def __init__(self):
+        es_conn = env.get_credential('ES_CONN')
```

Comment:
- thanks!
- I added the elastic search to the credential service and it seemed to work fine with get_credential on dev. Is that not the right way to do it for openFEC?
- Yep, good catch.